src/main/java/org/sciborgs1155/robot/Ports.java

```diff
@@ -1,6 +1,33 @@
 package org.sciborgs1155.robot;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public final class Ports {
+
+  public static Map<Integer, String> idToName;
+
+  static {
+    idToName = new HashMap();
+    // idToName.put(id, name);
+    idToName.put(6, "shooter top");
```

Comment:
We should test this properly works before merging. Someone can do this tomorrow (!)